doc/source/pbm-commands.rst

```diff
@@ -198,5 +393,71 @@ Shows the status of |PBM|. The output provides the following information:
 - |PITR| status
 - Valid time ranges for point-in-time recovery and the data size
   
+The command accepts the following flags:
+
+.. list-table:: 
+   :header-rows: 1
+   :widths: auto
+
+   * - Flag
+     - Description
+   * - ``-o``, ``--out=text``
+     - Shows the status as either plain text or a JSON object. Supported values: text, json
+   * - ``-s``, ``--sections=SECTIONS``
+     - Shows the status for the specified section. You can pass several flags to view the status for multiple sections. Supported values: cluster, pitr, running, backups. 
+   
+.. container:: toggle "JSON"
+
+   .. container:: header
+
+      **JSON output**
+
+   .. code-block:: javascript
+
+      {
+        "backups": {
+          "type": "FS",
+          "path": "<my-backup-dir>",
+          "snapshot": [
+             ...
+            {
+              "name": "<backup_name>",
+              "size": 3143396168,
+              "status": "done",
+              "completeTS": Timestamp,
+              "pbmVersion": "1.6.0"
+            },
+          ],
+          "pitrChunks": {
+            "pitrChunks": [
+               ...
+              {
+                "range": {
+                  "start": Timestamp,
+                  "end": Timestamp
+                }
+              },
+            ],
+            "size": 677901884
+          }
+        },
+        "cluster": [
+          {
+            "rs": "<replSet_name>",
+            "nodes": [
+              {
+                "host": "<replSet_name>/example.mongodb:27017",
+                "agent": "v1.6.0",
+                "ok": true
+              }
+            ]
+          }
+        ],
+        "pitr": {
+          "conf": true,
+          "run": true
```

Comment:
If there are some pitr related errors there is also would be a section `error` that contains a text error message.

Reply:
"pitr": {
    "conf": true,
    "run": false,
    "error": "2021-08-11T12:17:07.000+0000 E [rs1/localhost:27018] [pitr] init: catchup: no backup found after the restored 2021-08-11T12:02:41Z, a new backup is required to resume PITR"
  }